### PR TITLE
Update Bioschemas description

### PR DESCRIPTION
**Pull request by tsufz**
@meier-rene and @egonw; I updated the schemas with the missing description and inchikey tags. @meier-rene, I did not check the code running, I hope those small steps in Java programming do their job.

Refers to issue #96 and #206.

### CODE REVIEW

File: MassBank-Project/MassBank-lib/src/main/java/massbank/Record.java [hunk 1]
```diff
@@ -661,7 +661,13 @@ public String createRecordString() {
 		return sb.toString();
 	}
 	
-	public String createStructuredData() {
+	public String DESCRIPTION() {
+		StringBuilder sb = new StringBuilder();
+		sb.append("This MassBank record with Accession " + ACCESSION() + " contains the " +	MS_TYPE()" mass spectrum" + " of '" + RECORD_TITLE().get(0) + "' with the InChIkey '" + link.getKey("INCHIKEY") + "'."
```
Comment (sneumann) [inline MassBank-Project/MassBank-lib/src/main/java/massbank/Record.java:666]: This looks as if there is at least one "+" missing 
`" contains the " +	MS_TYPE()" mass spectrum"`
Yours, Steffen

**Comment by meier-rene** [comment]
I will take care of this this afternoon.

**Comment by tsufz** [comment]
@meier-rene, would be nice that we could report, that we are fully compliant in the next Bioschemas meeting.

**Comment by tsufz** [comment]
@egonw could you comment on @meier-rene question, please?

Best,
Tobias

**Pull request merged**